<launch>
    <node pkg="arx_r5_controller" type="R5Controller" name="remote_slave_l" output="screen">
        <param name="arm_control_type" value="remote_slave_v1"/>
        <param name="arm_can_id" value="can1"/>
        <param name="arm_end_type" value="0"/>

        <param name="sub_topic_name" value="/joint_information"/>
        <param name="pub_topic_ee_name_v1" value="/follow1_pos_back"/>
        <param name="pub_topic_joint_name_v1" value="/slave_joint_information"/>
    </node>

    <node pkg="arx_r5_controller" type="R5Controller" name="remote_slave_r" output="screen">
        <param name="arm_control_type" value="remote_slave_v1"/>
        <param name="arm_can_id" value="can3"/>
        <param name="arm_end_type" value="0"/>

        <param name="sub_topic_name" value="/joint_information2"/>
        <param name="pub_topic_ee_name_v1" value="/follow2_pos_back"/>
        <param name="pub_topic_joint_name_v1" value="/slave_joint_information2"/>
    </node>
</launch>